string ades_name
Observation[] samples
---
bool success 
